### PR TITLE
Add sanity management test and remove unused ones

### PR DESCRIPTION
**Pull request by noloerino**
A lot of management scripts existed to account for some jank in the old schema, which is now gone. This sanity checks everything except the section creation scripts (although those are arguably the most important ones). Check by running `python csm_web/manage.py test scheduler.tests.test_management`.

### CODE REVIEW

File: csm_web/scheduler/management/commands/checkintegrity.py [hunk 1]
```diff
@@ -44,124 +26,31 @@ def _err(self, msg):
...
-                "Found {} attendances where section does not match profile".format(
-                    qs.count()
-                )
+            if section.mentor is None:
+                self._err(f"Section {section} has no mentor")
+
+        mentors = Mentor.objects.select_related("section__spacetime")
+        emails = mentors.values_list("user__email", flat=True).distinct()
+        for email in emails:
+            spacetimes = Spacetime.objects.filter(
```
Comment (Svetlitski) [inline csm_web/scheduler/management/commands/checkintegrity.py]:
```suggestion
            spacetimes = Spacetime.objects.filter(section__mentor__user__email=email)
```

File: csm_web/scheduler/management/commands/checkintegrity.py [hunk 2]
```diff
@@ -44,124 +26,31 @@ def _err(self, msg):
...
-
-    def _check_attendance_integrities(self):
-        qs = Attendance.objects.exclude(section=F("attendee__section"))
-        if qs.count() > 0:
-            self._err(
-                "Found {} attendances where section does not match profile".format(
-                    qs.count()
-                )
+            if section.mentor is None:
+                self._err(f"Section {section} has no mentor")
```
Comment (Svetlitski) [inline csm_web/scheduler/management/commands/checkintegrity.py]: This is probably just me not being as familiar with the management command framework, but is `self._err` supposed to be defined somewhere?

File: csm_web/scheduler/management/commands/checkintegrity.py [hunk 3]
```diff
@@ -44,124 +26,31 @@ def _err(self, msg):
...
+
+        mentors = Mentor.objects.select_related("section__spacetime")
+        emails = mentors.values_list("user__email", flat=True).distinct()
+        for email in emails:
+            spacetimes = Spacetime.objects.filter(
+                pk__in=mentors.filter(user__email=email).values_list("section__spacetime", flat=True)
             )
+            # Naive n^2 algorithm since n is almost certainly < 2
+            n = len(spacetimes)
+            for i in range(n):
```
Comment (Svetlitski) [inline csm_web/scheduler/management/commands/checkintegrity.py]:
```suggestion
            for i, st in enumerate(spacetimes):
```

More pythonic way to iterate over a list when you still need the index

File: csm_web/scheduler/management/commands/checkintegrity.py [hunk 4]
```diff
@@ -44,124 +26,31 @@ def _err(self, msg):
...
+                st = spacetimes[i]
+                st_start = dt.datetime.combine(dt.date.today(), st.start_time)
+                st_end = st_start + st.duration
+                for j in range(i + 1, n):
```
Comment (Svetlitski) [inline csm_web/scheduler/management/commands/checkintegrity.py]:
```suggestion
                for st_2 in spacetimes[i+1:]:
```

If you *really* care about the overhead of creating new lists by slicing, see `itertools.islice`

File: csm_web/scheduler/tests/test_management.py [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+        ("export_section", ["demo_user@berkeley.edu"]),
+        ("triagedups", ["CS61A"]),
+    )
+
+    def setUp(self):
+        self.cs61a = CourseFactory.create(name="CS61A")
+        self.user = UserFactory.create(email="demo_user@berkeley.edu")
+        mentor = MentorFactory.create(user=self.user)
+        SectionFactory.create(course=self.cs61a, mentor=mentor)
+        # Create a bunch of random guys for the heck of it
```
Comment (Svetlitski) [inline csm_web/scheduler/tests/test_management.py]: Instead use 
```python3

for section in SectionFactory.create_batch(10, course=self.cs61a):
    StudentFactory.create(section=section)
```
`create_batch` is faster than many individual creations, which admittedly doesn't matter right now, but could really slow things down if we eventually have automated CI

File: csm_web/scheduler/management/commands/checkintegrity.py (hunk 2 above)
Comment (noloerino) [reply csm_web/scheduler/management/commands/checkintegrity.py]: I defined it on the class further above; it's just a convenience method for all errors to get printed out at the very end

File: csm_web/scheduler/management/commands/checkintegrity.py (hunk 1 above)
Comment (noloerino) [reply csm_web/scheduler/management/commands/checkintegrity.py]: oops, I initially was trying to see if I could group them by email with a single query and it just mutated into this

**Pull request merged**